assets/js/admin.js

```diff
@@ -41,7 +41,7 @@ jQuery(document).ready(function($) {
     // Show/hide field for specific switches
     $('[data-expand!=""]').each( function() {
         var $thisSwitch = $( this );
-        var $showFieldRow = $( '#' + $thisSwitch.attr( 'data-expand' ) ).closest( 'tr' );
+        var $showFieldRow = $( '#' + $thisSwitch.attr( 'data-expand' ).trim() ).closest( 'tr' );
```

Comment:
Found an issue with whitespace formatting 😮 corrected the issue but this will prevent it from happening again.